src/server/models/schools.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const mongoose = require("mongoose");
+
+const schoolSchema = new mongoose.Schema({
```

Comment:
man you are really index happy. none of these fields probably need indexes

Reply:
If we can perform searches against them should they be indexed? I figured it would improve query speed but I can remove them if you would like